Formula/confluent-platform.rb

```diff
@@ -11,14 +11,44 @@ class ConfluentPlatform < Formula
 
   conflicts_with "kafka", :because => "kafka also ships with identically named Kafka related executables"
 
+  patch :p0, :DATA
```

Comment:
This patch needs to be submitted upstream as a pull request before we can accept it in the formula.

Reply:
I want to send a pull request, but I couldn't find the repository. For example, the script has the variable "HUB_CLI_CLASSPATH", but no page is found when I google it. In addition, no repository on GitHub has the script.
cf. https://github.com/search?q=org%3Aconfluentinc+HUB_CLI_CLASSPATH&type=Code

I guess Confluent Inc. has the script internally and put it when it releases archives.

Reply:
You could try to use `inreplace` to change it instead of a patch? Do they have an issue tracker you can report this to if not a pull request?

Reply:
I've tried `inreplace` in the commit https://github.com/Homebrew/homebrew-core/pull/54858/commits/d2f2a8bb3503aec9bfb46663c3070088ec3c5c7e.

I also posted a message in the community Slack to ask how to send a patch:

![image](https://user-images.githubusercontent.com/508822/82733160-016a2e80-9d4d-11ea-85cd-84d7cb1474db.png)
https://confluentcommunity.slack.com/archives/C488525JT/p1590243987224100